lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.html

```diff
@@ -41,6 +41,7 @@
                             </mat-autocomplete>
                             <button
                                 id="adf-select-process-dropdown"
+                                [attr.aria-label]="'ADF_CLOUD_PROCESS_LIST.ADF_CLOUD_START_PROCESS.FORM.SELECT_PROCESS_DROPDOWN' | translate"
```

Comment:
you can use "title" here, and all screen readers work with that fine

Reply:
it was set to title, changed it to aria-label because of Michal's comment. Changing it back to title again.